eq-author/cypress/integration/authenticated/properties_spec.js

```diff
@@ -141,27 +157,26 @@ describe("Answer Properties", () => {
       });
       describe("Required", () => {
         const labelText = "Required";
-        it("Should default to off", () => {
-          findByLabel(labelText).should("not.be.checked");
-        });
         it("Can toggle 'Required' property on and off", () => {
           findByLabel(labelText)
+            .first()
+            .should("not.be.checked");
+          findByLabel(labelText)
+            .first()
             .click({ force: true })
             .should("be.checked");
           findByLabel(labelText)
+            .first()
             .click({ force: true })
             .should("not.be.checked");
         });
       });
       describe("Date format", () => {
         const labelText = "Date type";
-        it("Should show 'Date type' label", () => {
-          findByLabel(labelText);
-        });
+
         it("Should default to dd/mm/yyyy", () => {
+          findByLabel(labelText);
```

Comment:
What does this do when it doesn't have any chained functions?

Reply:
Good spot. I have removed this line